src/main/java/org/openstreetmap/atlas/checks/event/CheckFlagEvent.java

```diff
@@ -217,4 +219,35 @@ public String toString()
     {
         return this.toGeoJsonFeatureCollection().toString();
     }
+
+    public String asLineDelimitedGeoJsonFeatures()
+    {
+        return asLineDelimitedGeoJsonFeatures((jsonObject) ->
+        {
+        });
+    }
+
+    public String asLineDelimitedGeoJsonFeatures(final Consumer<JsonObject> jsonMutator)
+    {
+        final JsonObject flagGeoJsonFeature = flag.asGeoJsonFeature();
+        final JsonObject flagGeoJsonProperties = flagGeoJsonFeature.get("properties")
+                .getAsJsonObject();
+        flagGeoJsonProperties.addProperty("flag:check", getCheckName());
```

Comment:
If we did `getSimpleName`, that would not be getting the name of the check's class, it would be getting `CheckFlagEvent`. That actual name is passed around through various constructors in the code base, but ultimately it is set in BaseCheck.java:

```java
private final String name = this.getClass().getSimpleName();
```

So, that is the actual property, but it's passed around several times. That can be cleaned up, but it's unrelated to the objective of this PR.

Reply:
Got messed up. Thought this was "getClassName". This is good 👍